backend/src/constants.ts

```diff
@@ -1,6 +1,10 @@
+import { DateTime } from 'luxon';
+
 export type GithubRepo = { name: string; organization: string };
 export type GithubUser = { login: string; id: number };
 
 export const GithubRepos: { [key: string]: GithubRepo } = {
   IMMICH: { name: 'immich', organization: 'immich-app' },
 };
+
+export const IMMICH_INCEPTION_UNIX_TIME = DateTime.fromISO('2022-02-01T00:00:00Z').toMillis();
```

Comment:
Another thing I've done is just have a constant for `IMMICH_INCEPTION` that's the `DateTime` and then to `date.toWhateverYouNeed` in the method.

Reply:
Yea, I did the same originally and switched. Happy with whichever so I will switch to a DateTime